src/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "data-ingest-board",
-  "version": "1.3.10",
+  "version": "1.3.11",
   "private": true,
   "scripts": {
     "all": "npm-run-all --parallel dev css addons",
-    "dev": "next dev",
+    "dev": "next dev --port 3001",
```

Comment:
@libpitt I think this was accidentally pushed

Reply:
@maxsibilla  I didn't accidentally push it. But can revert it if you think that is better. Portal runs on default `3000`, and to have them running at the same time better this on another port; I always have to change it whenever I work on this project. Just committed it indefinitely.

Reply:
@libpitt Thats fair, it doesn't interfere with the docker production build and deployment so I am ok with this then